monai/tests/test_clinical_preprocessing.py

```
@@ -0,0 +1,144 @@
+
+    preprocess_dicom_series("dummy_path.dcm", "MR")
+
+    mock_pipeline.assert_called_once()
+    mock_transform.assert_called_once_with("dummy_path.dcm")
+
+
+def test_preprocess_dicom_series_invalid_modality():
+    """Test preprocess_dicom_series raises ValueError for unsupported modality."""
+    try:
+        preprocess_dicom_series("dummy_path.dcm", "PET")
+        assert False, "Should have raised ValueError"
```

Ruff (0.14.8)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)